reV/generation/generation.py

```diff
@@ -12,18 +12,24 @@
 import pandas as pd
 
 from reV.generation.base import BaseGen
-from reV.utilities.exceptions import (ProjectPointsValueError, InputError)
-from reV.SAM.generation import (Geothermal,
-                                PvWattsv5,
-                                PvWattsv7,
-                                PvWattsv8,
-                                PvSamv1,
-                                TcsMoltenSalt,
-                                WindPower,
-                                SolarWaterHeat,
-                                TroughPhysicalHeat,
-                                LinearDirectSteam,
-                                MhkWave)
+from reV.utilities.exceptions import (
```

Comment:
Can we retain original formatting here and anywhere else in the PR where a pure formatting change was made?